pkg/graph/schema.graphql

```diff
@@ -1610,6 +1716,13 @@ type TRBRequestForm {
   collabDateGovernanceReviewBoard: Time
   collabDateOther: Time
   collabGroupOther: String
+  subjectAreaTechnicalReferenceArchitecture: TRBTechnicalReferenceArchitectureOption
+  subjectAreaNetworkAndSecurity: TRBNetworkAndSecurityOption
+  subjectAreaCloudAndInfrastructure: TRBCloudAndInfrastructureOption
+  subjectAreaApplicationDevelopment: TRBApplicationDevelopmentOption
+  subjectAreaDataAndDataManagement: TRBDataAndDataManagementOption
+  subjectAreaGovernmentProcessesAndPolicies: TRBGovernmentProcessesAndPoliciesOption
+  subjectAreaOtherTechnicalTopics: TRBOtherTechnicalTopicsOption
```

Comment:
Should these all be arrays? [The figma](https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?node-id=10%3A3661) shows all of these as multi-selects, so I think the intent is to be able to select any number of these for each option.

Reply:
Whoops, didn't realize they were multi selects. I made that change.